android/app/build.gradle

```diff
@@ -111,7 +111,8 @@ def enableProguardInReleaseBuilds = false
  * give correct results when using with locales other than en-US.  Note that
  * this variant is about 6MiB larger per architecture than default.
  */
-def jscFlavor = 'org.webkit:android-jsc:+'
+// def jscFlavor = 'org.webkit:android-jsc:+'
+def jscFlavor = 'org.webkit:android-jsc-intl:+'
```

Comment:
Just curios, why do we need to change this config?

Reply:
Please remove the code comment also.

Reply:
When i run android, it's has a bug, Intl is undefined
<img width="856" alt="Screen Shot 2022-02-19 at 10 50 22" src="https://user-images.githubusercontent.com/34257375/154785018-4aeb0d04-1e51-49f5-8aa2-8db2b451d287.png">
.